docs/tutorials/MoneyModel.py

```diff
@@ -62,12 +62,13 @@ def __init__(self, n=10, width=10, height=10, seed=None):
         self.grid = mesa.space.MultiGrid(width, height, True)
 
         # Create agents
-        for _ in range(self.num_agents):
-            a = MoneyAgent(self)
+        agents = MoneyAgent.create_agents(model=self,n=n)
```

Comment:
It’s best practice to leave a space after each argument when calling a method. Please update this everywhere.
```suggestion
        agents = MoneyAgent.create_agents(model=self, n=n)
```

Reply:
I will format these according to black formatter